src/Struct/FindologicProduct.php

```diff
@@ -748,14 +748,24 @@ protected function setOrdernumberByProduct(ProductEntity $product): void
     protected function setCategoriesAndCatUrls(): void
     {
         $productCategories = $this->product->getCategories();
-        if ($productCategories === null || empty($productCategories->count())) {
+        $productVariants = $this->product->getChildren()->getElements();
+
+        if (($productCategories === null || empty($productCategories->count()))) {
```

Comment:
This condition is inside of the bracket because I had one more condition here to add.